src/modcdfnames_CMIP6.h90

```diff
@@ -0,0 +1,153 @@
+  CHARACTER(LEN=256) :: cn_vlat1d  = 'lat'            !: latitude  1d
+
+  ! Attribute of a variable
+  CHARACTER(LEN=256) :: cn_missing_value = '_FillValue' !: missing value (to be replaced bby _Fill_Value)
+
+  ! Metrics
+  CHARACTER(LEN=256) :: cn_ve1t='e1t', cn_ve2t='e2t'   !: e.t
+  CHARACTER(LEN=256) :: cn_ve1u='e1u', cn_ve2u='e2u'   !: e.u
+  CHARACTER(LEN=256) :: cn_ve1v='e1v', cn_ve2v='e2v'   !: e.v
+  CHARACTER(LEN=256) :: cn_ve1f='e1f', cn_ve2f='e2f'   !: e.v
+  CHARACTER(LEN=256) :: cn_ve3t='e3t_0', cn_ve3w='e3w_0'   !: e3.
+  CHARACTER(LEN=256) :: cn_ve3v='e3v_0', cn_ve3u='e3u_0'   !: e3.
+  CHARACTER(LEN=256) :: cn_vff='ff'
+
+  CHARACTER(LEN=256) :: cn_gdept='gdept_1d', cn_gdepw='gdepw_1d' !: 1d dep variable
+  CHARACTER(LEN=256) :: cn_hdept='hdept'   , cn_hdepw='hdepw'    !: 2d dep variable
+
+  CHARACTER(LEN=256) :: cn_glamt='glamt', cn_gphit='gphit'   !:  glam gphi
+  CHARACTER(LEN=256) :: cn_glamu='glamu', cn_gphiu='gphiu'   !:  glam gphi
+  CHARACTER(LEN=256) :: cn_glamv='glamv', cn_gphiv='gphiv'   !:  glam gphi
+  CHARACTER(LEN=256) :: cn_glamf='glamf', cn_gphif='gphif'   !:  glam gphi
+
+  ! Generic mesh-mask file names  cn_f...
+  CHARACTER(LEN=256) :: cn_fzgr='mesh_mask.nc'
+  CHARACTER(LEN=256) :: cn_fhgr='mesh_mask.nc'
+  CHARACTER(LEN=256) :: cn_fmsk='mesh_mask.nc'
+  CHARACTER(LEN=256) :: cn_fcoo='coordinates.nc'
+  CHARACTER(LEN=256) :: cn_fbasins='subbasin.nc'
+
+  ! msk dimension
+  CHARACTER(LEN=256) :: cn_coox='x'          !: longitude, I dimension
+  CHARACTER(LEN=256) :: cn_cooy='y'          !: latitude,  J dimension
+
+  ! Mask variable
+  CHARACTER(LEN=256) :: cn_vatlmsk='atlmsk'
+  CHARACTER(LEN=256) :: cn_vindmsk='indmsk'
+  CHARACTER(LEN=256) :: cn_vpacmsk='pacmsk'  
+
+  ! Variable name  : cn_v... [ starts with cn_v ]
+  CHARACTER(LEN=256) :: cn_votemper='thetao'   !: temperature
+  CHARACTER(LEN=256) :: cn_vosaline='so'       !: salinity
+  CHARACTER(LEN=256) :: cn_votemper2='thetao_sqd' !: squared temperature
+  CHARACTER(LEN=256) :: cn_vosaline2='so_sqd'  !: squared salinity
+  CHARACTER(LEN=256) :: cn_vozocrtx='uo'       !: zonal velocity
+  CHARACTER(LEN=256) :: cn_vomecrty='vo'       !: meridional velocity
+  CHARACTER(LEN=256) :: cn_vozocrtx2='u2o'     !: squared velocity
+  CHARACTER(LEN=256) :: cn_vomecrty2='v2o'     !: squared velocity
+  CHARACTER(LEN=256) :: cn_vomeeivv='vomeeivv' !: meridional Eddy Induced Velocity ! NOT A CMIP6 VARIABLE (only eiv transport asked)
+  CHARACTER(LEN=256) :: cn_vovecrtz='wo'       !: vertical velocity
+  CHARACTER(LEN=256) :: cn_vovecrtz2='w2o'     !: squared vertical velocity
+  CHARACTER(LEN=256) :: cn_sossheig='zos'      !: Dynamic Sea Surface Height
+  CHARACTER(LEN=256) :: cn_sossheig2='zossq'   !: Squared Dynamic Sea Surface Height
+  CHARACTER(LEN=256) :: cn_somxldep='mlotst'   !: Mixed layer depth
+  CHARACTER(LEN=256) :: cn_somxl010='somxl010' !: Mixed layer depth (density criterium)      
```

Comment:
It is a very minor point, but it might be useful to clarify which is the 'diagnostic' mixed layer depth variable. One might vary the MLD for which they calculate the heat content, but it is perhaps not clear which variable should be changed. 

It is a minor point as it is easy to check the source code, but would be useful to have in the 'docstring'.